src/Mod/TechDraw/App/Preferences.cpp

```diff
@@ -534,21 +534,16 @@ std::string Preferences::currentElementDefFile()
 int Preferences::LineCapStyle()
 {
     int currentIndex = LineCapIndex();
-    int result{0x20};
-        switch (currentIndex) {
+    switch (currentIndex) {
         case 0:
-            result = static_cast<Qt::PenCapStyle>(0x20);   //round;
-            break;
+            return static_cast<Qt::PenCapStyle>(0x20);   //round;
```

Comment:
Offtopic: Is there any reason to use magic numbers instead of `Qt::FlatCap`, `Qt::SquareCap` and `Qt::RoundCap`?